docs/source/notebook-components/weaver_example.ipynb

```diff
@@ -10,25 +10,40 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.    if status not in body:
</pre><p>Incorrect check. "status" (the property name) must be in <code>body</code>. </p><p>Currently,  <code>""</code> (initialized) or whichever value that is obtained will always succeed <code>not in body</code> check. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/359/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>